### DISCUSSION TITLE
Need Help with Loading Issues in Health Care AI Project Deployed on Render

### DISCUSSION THREAD

**Discussion by rameshgyawali127** [community/community#126075] [Programming Help]
### Select Topic Area

Question

### Body

Hello everyone,

I've deployed a healthcare AI project on Render, but I'm encountering loading issues with the web application. Sometimes the page takes an extended period to load, and occasionally it fails to load altogether.

The project is built using machine learning models and Flask for the web interface. I've tried to diagnose the issue by optimizing code and reducing dependencies, but the problem persists.

Could anyone offer guidance on troubleshooting and resolving loading issues in Flask applications deployed on Render? Are there any common pitfalls or configurations that might be causing these delays or failures?

I'm open to suggestions for optimizing performance, implementing caching strategies, or any other techniques that could improve the loading speed and reliability of the application.

Any insights or advice from those experienced with deploying Flask applications on Render would be greatly appreciated.

Thank you for your time and assistance.

Best regards,
Ramesh Gyawali

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @rameshgyawali127 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by CopperEagle** [comment]
Hey @rameshgyawali127 

It's hard to say much from afar, without code and loading diagram. How about you have a look at the methodologies [here](https://scoutapm.com/blog/improving-flask-performance) specifically for flask? It talks though a lot of things, including analysis and caching.

**Reply by rameshgyawali127** [reply]
Once see my code if anything I did wrong just fork  in my repo it or mentioned it? "https://github.com/rameshgyawali127/HealthCare-Recommendation-AI"
Anyway, Thanks for your help.
